### PR TITLE
Polishing first part of recurring api guide

### PR DESCRIPTION
**Pull request by rebeksburnett**
The only thing that is not trivial is in the introduction section. See it rendered here: https://github.com/vippsas/vipps-recurring-api/blob/2687f70ede4ed65d00e2483d1c3c25dfe7e8fcd2/vipps-recurring-api.md

### CODE REVIEW

**Comment by rebeksburnett** [comment]
@flinkgutt I got some clarification from Thomas T. See what you think now. :-)

File: vipps-recurring-api.md [hunk 1]
```diff
@@ -458,42 +451,43 @@ Example for a subscription every 30th day:
...
-[Campaigns](#campaigns) in combination with initial charge
-if the subscription is cheaper in the beginning than the normal price later.
+**Please note:** If the subscription is cheaper in the beginning than the normal price later, use
+[campaigns](#campaigns) in combination with initial charge.
 If you use `initialcharge` alone for campaigns, users will be confused by how it appears in Vipps,
 as it looks like the full price period starts immediately.
 
 Initial charge will be performed if the `initialcharge` is provided when
-creating an agreement. If there is no initial charge: Don't sent `initialcharge`
+creating an agreement. If there is no initial charge, don't sent `initialcharge`
```
Comment (ameliemaucade) [inline vipps-recurring-api.md:468]: Tiny old typo here. Should be `don't send`

**Review by ameliemaucade — CHANGES_REQUESTED** [review]
Two small comments, but other than that it's great

File: vipps-recurring-api.md [hunk 2]
```diff
@@ -2,38 +2,37 @@
...
 that will be automatically processed on the due date.
 
-The overall idea is:
-* Merchants let users enter agreements in Vipps.
-* Merchants ask Vipps to make charges, and Vipps handles _everything_.
-* Users have the full overview in Vipps, including a link to the merchant's website.
+The overall flow is:
+* The merchant creates a draft agreement and proposes it to the customer via Vipps.
+* The customer approves the agreement in Vipps.
+* At the due date, the merchant sends a charge request to Vipps.
```
[the remark this answers is not in the document]
Comment (rebeksburnett) [reply vipps-recurring-api.md:11]: Yes, thank you!

File: vipps-recurring-api.md (hunk 1 above)
Comment (rebeksburnett) [reply vipps-recurring-api.md]: Oh! I was blind to that one. It took a minute to see it. :')

**Comment by rebeksburnett** [comment]
Those should be fixed now. :-)

**Review by ameliemaucade — APPROVED** [review]
👍

**Pull request merged**